src/components/side_panel/conditional_formatting/conditional_formatting.ts

```diff
@@ -446,9 +442,10 @@ export class ConditionalFormattingPanel extends Component<Props, SpreadsheetChil
 
   saveConditionalFormat() {
     if (this.state.currentCF) {
-      const invalidRanges = this.state.currentCF.ranges.some((xc) => !xc.match(rangeReference));
-      if (invalidRanges) {
-        this.state.errors = [CommandResult.InvalidRange];
+      if (
+        this.state.errors.includes(CommandResult.EmptyRange) ||
+        this.state.errors.includes(CommandResult.InvalidRange)
+      ) {
```

Comment:
```suggestion
      if (this.state.errors.lenght) {
```
is there a situation where we want to allow saving if there is an error somewhere ?

Reply:
Then you'd try to save a CF with invalid arguments -> you have some errors because you forgot some argument, w/e. and since you have an errro ,you can't save anymore ^^you're stuck :p